core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -589,6 +589,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey UNDERFS_OBJECT_STORE_MULTI_RANGE_CHUNK_SIZE_BYTES =
+      new Builder(Name.UNDERFS_OBJECT_STORE_MULTI_RANGE_CHUNK_SIZE_BYTES)
+          .setDefaultValue(String.format("${%s}", Name.USER_BLOCK_SIZE_BYTES_DEFAULT))
```

Comment:
Our block size is pretty large, how does a smaller size (say 32, 64, or 128 MB) affect performance? I think we do this as a solution to object stores which do not offer an abort api, meaning we will read the entire range even if the stream is closed early.

Reply:
Previously we used the user block size which is why this is set as the default. I don't know what the performance implications are. But this is preserving the current functionality. If we want to investigate performance it will require some testing.

Reply:
correct, that is the reason. 
+1 for smaller size. the performance impact would be workload dependent. 32MB seems big enough not to impact full scans such as wordcount adversely.

Reply:
Okay, I'm fine with preserving the previous value and opening a separate ticket for investigating this.